RuntimeException: An error occured while executing doInBackground() Caused by: ExceptionInInitializerError WallabagConnection.getClient()

already reported 5 times in Gplay:

```
java.lang.RuntimeException: An error occured while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:300)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:841)
Caused by: java.lang.ExceptionInInitializerError
	at fr.gaulupeau.apps.Poche.network.WallabagConnection.getClient(WallabagConnection.java:107)
	at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.getInputStream(UpdateFeedTask.java:248)
	at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.updateByFeed(UpdateFeedTask.java:190)
	at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.update(UpdateFeedTask.java:172)
	at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.doInBackground(UpdateFeedTask.java:93)
	at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.doInBackground(UpdateFeedTask.java:32)
	at android.os.AsyncTask$2.call(AsyncTask.java:288)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 4 more
Caused by: java.lang.NullPointerException
	at okhttp3.internal.tls.RealTrustRootIndex.<init>(RealTrustRootIndex.java:31)
	at okhttp3.internal.Platform.trustRootIndex(Platform.java:100)
	at okhttp3.internal.Platform$Android.trustRootIndex(Platform.java:249)
	at okhttp3.OkHttpClient.<init>(OkHttpClient.java:189)
	at okhttp3.OkHttpClient.<init>(OkHttpClient.java:60)
	at okhttp3.OkHttpClient$Builder.build(OkHttpClient.java:718)
	at fr.gaulupeau.apps.Poche.network.WallabagConnection.createClient(WallabagConnection.java:175)
	at fr.gaulupeau.apps.Poche.network.WallabagConnection.getClient(WallabagConnection.java:110)
	at fr.gaulupeau.apps.Poche.network.WallabagConnection$Holder.<clinit>(WallabagConnection.java:179)
	... 12 more
```


```
vor 47 Minuten
	
1.9.9.4
	
Aquaris E10 (Aquaris_E10)
	
fallo cuando sincronizo
```


```
Gestern, 21:34
	
1.9.9.4
	
Galaxy Tab Pro 8.4 (mondrianwifiue)
	
it crashes while syncing
```

Android 4.4 	3
Android 6.0 	2

--- a/app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagConnection.java
+++ b/app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagConnection.java
@@ -13,7 +13,6 @@ import okhttp3.HttpUrl;
 import okhttp3.Interceptor;
 import okhttp3.JavaNetCookieJar;
 import okhttp3.OkHttpClient;
-import okhttp3.Protocol;
 import okhttp3.Request;
 import okhttp3.Response;
 
@@ -25,7 +24,6 @@ import java.net.Socket;
 import java.security.KeyManagementException;
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.LinkedList;
@@ -143,7 +141,7 @@ public class WallabagConnection {
 
                             @Override
                             public java.security.cert.X509Certificate[] getAcceptedIssuers() {
-                                return null;
+                                return new java.security.cert.X509Certificate[0];
                             }
                         }
                 };

